lib/mercenary/presenter.rb

```diff
@@ -31,8 +31,9 @@ def options_presentation
 
     def command_options_presentation
       return nil if command.options.empty?
-
-      command.options.map(&:to_s).join("\n")
+      options = command.options
+      options -= command.parent.options unless command.root?
```

Comment:
I do not see the need for calling a wrapper method here since you can call `command.parent.nil?` directly just as easily.
In fact, you can even remove the nil-check entirely by testing for "truthy" instead:
```ruby
options -= command.parent.options if command.parent
```

Reply:
Fixed.